src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -462,7 +462,7 @@ private boolean exportMatches(List<ParserResult> loaded) {
         try {
             matches = new DatabaseSearcher(query, databaseContext, preferencesService.getFilePreferences()).getMatches();
         } catch (IOException e) {
-            System.err.println(Localization.lang("Error occurred when searching") + ": " + e.getLocalizedMessage());
+            LOGGER.error("Error occurred when searching", e);
```

Comment:
Logger won't work with cli I think

Reply:
Please check the code 20 lines above. 

```java
            } else {
                LOGGER.error("Skipped - PDF {} does not exist", fileName);
            }
```

I checked it. 

My review comment was under the assumption that the risk that the exception was thrown is equally high than the other IOExceptions

I also know that tinylog outputs to the console in the case of CLI.